vic/drivers/image/src/vic_image.c

```diff
@@ -131,9 +131,6 @@ main(int    argc,
     // start vic run timer
     timer_start(&(global_timers[TIMER_VIC_RUN]));
 
-    timer_init(&(global_timers[TIMER_VIC_FORCE]));
-    timer_init(&(global_timers[TIMER_VIC_WRITE]));
```

Comment:
Why do we not have to initialize timers? If that is done as part of `timer_start()`, then we can also remove `timer_init()` from:
* `vic/drivers/shared_all/include/vic_driver_shared_all.h`
* `vic/drivers/shared_all/src/timing.c`
* `tests/unit/shared/test_timing.py` (`vic_lib.timer_init(timer)`)

Reply:
Actually `timer_init()` is called from `timer_start()`, so you do not have to call it separately, but you cannot delete it from the other files as I suggest above, so this should be OK.

Reply:
We could combine `timer_init()` and `timer_start()`, but just leave as is for now.

Reply:
Just to be clear, the reason we have separate `timer_init` and `timer_start` functions is for the CESM driver case where the `vic_cesm_run` function needs to "continue" a previously initialized timer. I don't think the image driver has a use for the separate functions. And yes, `timer_start = timer_init + timer_continue`.